llvm/test/MC/AArch64/SVE2/directive-arch_extension.s

```diff
@@ -19,3 +19,17 @@ rax1 z0.d, z0.d, z0.d
 .arch_extension sve2-bitperm
 bgrp z21.s, z10.s, z21.s
 // CHECK: bgrp z21.s, z10.s, z21.s
+
+.arch_extension sve2
```

Comment:
We could omit the `.arch_extension sve2` from the tests below, it follows from line 3.